### DISCUSSION TITLE
SIM based call

### DISCUSSION THREAD

**Discussion by ababila** [Salamek/huawei-lte-api#191] [Q&A] [answered]
Is possible to make SIM based calls with this project where the sent/received audio flow of the call can be for example read/write form/to a file ? 

(If yes will share some pointers to get stated please ?)

**Comment by Salamek** [comment]
If your router can't do it, this project can't either

**Comment by ababila** [comment]
Not using router, I mean using a USB Dongle like Huawei E3131 which is listed in the readme

**Reply by Salamek** [reply, marked as answer]
That dongle is still a router with virtual Ethernet interface via USB and WebUI.  You can switch it into serial mode and use PPPoE to control it but that has nothing to do with this lib.

**Comment by ababila** [comment]
okay. Thank you
